camelot/view/controls/editors/customeditor.py

```diff
@@ -180,3 +217,34 @@ def sizeHint(self):
             size_hint.setWidth(max(size_hint.width(), self.size_hint_width))
         return size_hint
 
+    def get_validator(self, validator_type, parent=None):
+        from vfinance.view import validators
```

Comment:
hier mag wel geen vf import staan

mss beter in de abstract validator ook een name registry steken ...